lib/components/tabs.vue

```diff
@@ -132,6 +136,14 @@
             },
             navStyle() {
                 return this.pills ? 'pills' : 'tabs';
+            },
+            computedPosition() {
+                if (this.bottom) {
+                    console.warn('Using the bottom property is deprecated and will be removed in a future version, use position="bottom" instead')
```

Comment:
use the `/utils/warn` helper method here,  See comment in the main pull request thread.